authentication-wrapper/src/main/java/io/mosip/idp/authwrapper/service/MockAuthenticationService.java

```diff
@@ -115,19 +130,28 @@ public ResponseWrapper<KycAuthResponse> doKycAuth(@NotBlank String licenseKey, @
             return responseWrapper;
         }
 
-        String kycToken = getKycToken(kycAuthRequest.getIndividualId(), clientId, relyingPartyId);
+        String psut;
+        try {
+            psut = IdentityProviderUtil.generateB64EncodedHash(ALGO_SHA3_256,
+                    String.format(PSUT_FORMAT, kycAuthRequest.getIndividualId(), relyingPartyId));
+        } catch (IdPException e) {
+            responseWrapper.getErrors().add(new Error("mock-ida-006", "Failed to generate PSUT"));
+            return responseWrapper;
+        }
+        String kycToken = getKycToken(kycAuthRequest.getIndividualId(), clientId, relyingPartyId, psut);
         KycAuthResponse kycAuthResponse = new KycAuthResponse();
         kycAuthResponse.setKycToken(kycToken);
-        kycAuthResponse.setUserAuthToken(UUID.randomUUID().toString());
+        kycAuthResponse.setUserAuthToken(psut);
```

Comment:
The naming of userAuthToken is confusing, can we rename idp core request and response classes to call it PartnerSpecificUserToken ?